chainer_computational_cost/computational_cost_hook.py

```diff
@@ -437,33 +476,47 @@ def _show_report_body(self, report, ost, mode, unit, cols,
                              "Available options: {}"
                              .format(cols, ", ".join(rep.keys())))
 
-        if unit not in self._flops_coeff_table:
-            raise ValueError("Please specify either None, 'K', 'M', 'G' or 'T'"
-                             " to argument `unit`.")
-        coeff_flops = self._flops_coeff_table[unit]
-        coeff_bytes = self._bytes_coeff_table[unit]
         if unit is None:
             unit = ''
+        if unit not in self._unit_list + ['autoaligned', 'auto']:
+            raise ValueError("Please specify either '', 'K', 'M', 'G', 'T',"
+                             "`autoaligned`  or 'auto' to argument `unit`.")
```

Comment:
So sorry for the late response :bow:
It's too trivial but, the output from this line is like:
```
ValueError: Please specify either '', 'K', 'M', 'G', 'T',`autoaligned`  or 'auto' to argument `unit`.
                                                         ^            ^
```

Reply:
Thank you for your pointing out.
I fixed it.